### PR TITLE
feat: add origin src to inputs

### PR DESCRIPTION
**Pull request by zongUMR**
`src` will be added `absolute path`, I add another field to save the original path(src and dist), so we can show the current info on website:
Screen Shot 2020-10-21 at 10 17 00

### CODE REVIEW

**Comment by zongUMR** [comment]
@yugasun Please review this PR

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [3.2.5](https://github.com/serverless/components/compare/v3.2.4...v3.2.5) (2020-10-21)
+
+- Feat: Add original src in inputs
```
Comment (timqian) [inline CHANGELOG.md:7]: This feature has not been released as well, maybe add it here as well? https://github.com/serverless/components/pull/823

**Comment by yugasun** [comment]
Well done, I have another question, when using `getInstance` to get instance detail, `inputs.src` of detail will be revert to `originSrc` or both `src` and `originSrc` will exist?

File: CHANGELOG.md (hunk 1 above)
Comment (zongUMR) [reply CHANGELOG.md:7]: Your meaning is that part of code was merge into `master` but do not want to release?

**Comment by zongUMR** [comment]
> Well done, I have another question, when using `getInstance` to get instance detail, `inputs.src` of detail will be restored to `originSrc` or both `src` and `originSrc` will exist?

They will be both saved

File: CHANGELOG.md (hunk 1 above)
Comment (timqian) [reply CHANGELOG.md:7]: Yes~

**Review by timqian — APPROVED** [review]

**Comment by yugasun** [comment]
So when web page try to display the result of deployment, the `src` will still be absolute path?

**Comment by zongUMR** [comment]
> So when web page try to display the result of deployment, the `src` will still be absolute path?

Yes, I do not change the old logic for `src` to avoid any problem from this field, just add a new one

**Comment by yugasun** [comment]
Well, maybe the frontend can try to restore `src` to `originSrc`.

**Pull request merged**